suite-common/wallet-core/src/accounts/accountsReducer.ts

```diff
@@ -196,7 +196,10 @@ export const selectDeviceAccountsByNetworkSymbol = memoizeWithArgs(
 
         const accounts = selectDeviceAccounts(state);
 
-        return A.filter(accounts, account => account.symbol === networkSymbol);
+        return A.filter(
+            accounts,
+            account => account.symbol === networkSymbol && (!account.empty || account.visible),
```

Comment:
I don't think this is good approach, if you need some specific derivation of Accounts (like `!empty || visible`) please create selector just for that. We will need to select empty accounts.